modules/serverless-create-kafka-broker-custom-yaml.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+//  * serverless/knative_eventing/serverless-using-brokers.adoc
+
+[id="serverless-create-kafka-broker-custom-yaml_{context}"]
+= Creating a Kafka broker with custom configuration by using YAML
+
+You can create a Kafka broker by using YAML with custom configuration.
+
+:FeatureName: Creating a Kafka broker with custom configuration
```

Comment:
I am not 100% I do understand the suggestion :see_no_evil: 

Is my doc wrong "package"?

Reply:
@matzew basically you cannot use an `include` here to add the technology preview bit because this file is already a module, i.e. ` modules/serverless-create-kafka-broker-custom-yaml.adoc`
The comment you have included at the top of the file:

```
// Module included in the following assemblies:
//
//  * serverless/knative_eventing/serverless-using-brokers.adoc
```

refers to this. The content here https://github.com/openshift/openshift-docs/pull/39408/files#diff-f77d942aa26ea42173a302709c69c3bde6c8f81aa2d202a8435b717965f92330R27-R37 is including files in an assembly, as an example.

Honestly I'm not sure about how this has been structured in https://github.com/openshift/openshift-docs/pull/39408/files#diff-f77d942aa26ea42173a302709c69c3bde6c8f81aa2d202a8435b717965f92330R27-R37 - this is why I would prefer that engineering just provide content, and then PRs are raised by docs folks who are aware of all the formatting and structure requirements and limitations. It's sort of making more work for us if we have to figure out how to make this PR work than it would be if the work was just scoped properly for the docs team for a release.

Reply:
OK, thanks! Let me see tomorrow howto re-structure this a bit differently. 

Otherwise we can close the PR and use it as a reference for content provided by engineering, for a more proper docs PR.